celery/backends/dynamodb.py

```diff
@@ -194,6 +222,17 @@ def _get_or_create_table(self):
                     self.table_name
                 )
             )
+            # Enable time-to-live on the table, ignoring whether or not TTL is
+            # currently specified as an option for the backend. This allows
+            # enabling it later. Until then, items are inserted without a value
+            # in the ttl field, meaning that DynamoDB will never expire them.
+            self._client.update_time_to_live(**self._get_ttl_specification())
```

Comment:
TTL configuration should be applied conditionally, if `time_to_live_seconds` is not `None`.

Reply:
Addressed.